rental.rb

```diff
@@ -1,10 +1,11 @@
 class Rental
-  attr_accessor :date, :book, :person
+  attr_accessor :date
+  attr_reader :book, :person
 
-  def initialize(book, date, person)
+  def initialize(date, book, person)
     @date = date
 
-    @person = perosn
+    @person = person
     person.rentals << self
 
```

Comment:
When I try to create a rental, I get an error, which reads that method rental, is not defined in the person class.

For reference, please do refer to, book.rb file.

Reply:
Thanks for your review, I have worked on it.